index.html

```diff
@@ -11909,6 +11909,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>Authors SHOULD NOT use <pref>aria-placeholder</pref> instead of a label as their purposes are different: The label indicates what kind of information is expected. The placeholder text is a hint about the expected value. See related <pref>aria-labelledby</pref> and <pref>aria-label</pref>.</p>
 				<p>Authors SHOULD present this hint to the user by displaying the hint text at any time the control's <span>value</span> is the empty string. This includes cases where the control first receives focus, and when users remove a previously-entered value.</p>
 				<p class="note">As is the case with the related <code>placeholder</code> attribute in [[HTML]], use of placeholder text as a replacement for a displayed label can reduce the accessibility and usability of the control for a range of users including older users and users with cognitive, mobility, fine motor skill or vision impairments. While the hint given by the control's label is shown at all times, the short hint given in the placeholder attribute is only shown before the user enters a value. Furthermore, placeholder text may be mistaken for a pre-filled value, and as commonly implemented the default color of the placeholder text provides insufficient contrast and the lack of a separate visible label reduces the size of the hit region available for setting focus on the control.</p>
+				<p class="note">The following examples could use the basic <code>&lt;label for="birthday"&gt;</code>/<code>&lt;input id="birthday"&gt;</code> construction, but here the <code>contenteditable</code> area does not allow for <code>label</code>. Association is thus achieved via <code>aria-labelledby</code>.</p>
```

Comment:
Adding an extra piece of example code seems confusing. Please simplify. Perhaps - "the following examples do not use the HTML label element as it cannot be used to label HTML elements with contenteditable."